src/steth/scWETH.sol

```diff
@@ -80,7 +81,7 @@ contract scWETH is sc4626, IFlashLoanRecipient {
     }
 
     constructor(ConstructorParams memory _params)
-        sc4626(_params.admin, _params.weth, "Sandclock WETH Vault", "scWETH")
+        sc4626(_params.admin, _params.keeper, _params.weth, "Sandclock WETH Vault", "scWETH")
     {
         if (_params.admin == address(0)) revert ZeroAddress();
```

Comment:
```suggestion
        if (_params.admin == address(0)) revert ZeroAddress();
        if (_params.keeper== address(0)) revert ZeroAddress();
```

Reply:
Just to be extra safe.

Reply:
we should move these checks to sc4626.sol

Reply:
fixed [here](https://github.com/lindy-labs/sandclock-contracts/pull/45/commits/83f382b67556acddcad66a82a3f487693d5af1ab)